tests/helper/helper_dev.go

```diff
@@ -105,10 +127,23 @@ func (o DevSession) Stop() {
 	o.session.Interrupt()
 }
 
+func (o DevSession) WaitSync() ([]byte, []byte) {
+	WaitForOutputToContain("Pushing files...", 180, 10, o.session)
+	WaitForOutputToContain("Waiting for something to change", 180, 10, o.session)
```

Comment:
This message will change once #5526 gets merged. Just FYI.

Reply:
As this message is not only informational but used for tests to work correctly, I think it would be nice to have a constant defined in the source. I'll add it once #5526 is merged

Reply:
Sounds good. 👍🏾